src/app.module.ts

```diff
@@ -0,0 +1,32 @@
+import { Module } from '@nestjs/common';
+import { HealthController } from './health/health.controller';
+import { OrdersModule } from './orders/orders.module';
+import { ProductsModule } from './products/products.module';
+import { CustomersModule } from './customers/customers.module';
+import { SharedModule } from './shared/shared.module';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { Customer } from './customers/domain/customer.domain';
+
+@Module({
+  imports: [
+    OrdersModule,
+    ProductsModule,
+    CustomersModule,
+    SharedModule,
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: 'localhost',
+      port: 5432,
+      username: 'msgcsuser',
+      password: 'msgcspass',
+      database: 'msgcsdb',
+      entities: [Customer],
```

Comment:
Here you have to specify which `entities` TypeORM should manage. Does the creation of tables and so on work without this?